terraform/aks.tf

```diff
@@ -21,14 +22,15 @@ resource "azurerm_kubernetes_cluster" "example" {
 }
 
 output "client_certificate" {
-  value = azurerm_kubernetes_cluster.example.kube_config.0.client_certificate
+  value = var.existing_pachyderm_cluster_name == "" ? azurerm_kubernetes_cluster.example[0].kube_config.0.client_certificate : ""
```

Comment:
The syncer does need a client certificate for an existing cluster, no? Does it get passed in elsewhere?